RedisEntityListenerTest.onSlowPublish failed in github action

<!--  Thanks for submitting a bug report!  Before submitting:
1. Try searching the existing issues to see if your issue has already been reported
2. Try asking on our Discord group (https://discordapp.com/invite/FFb9YFX)
3. If you're reporting a security vulnerability, please email security@hedera.com instead of opening an issue
-->

**Detailed Description**

https://github.com/hashgraph/hedera-mirror-node/pull/1648/checks?check_run_id=2007594434

```
2021-03-01T20:57:00.5249582Z [INFO] Running com.hedera.mirror.importer.parser.record.entity.redis.RedisEntityListenerTest
2021-03-01T20:57:00.5917654Z 2021-0***-01T14:57:00.587-0600 WARN Thread-56 c.h.m.i.p.r.e.r.RedisEntityListener topicMessagesQueue is full, will block until space is available 
2021-03-01T20:57:00.6270920Z 2021-0***-01T14:57:00.609-0600 INFO pool-28-thread-1 c.h.m.i.p.r.e.r.RedisEntityListener Finished notifying 1 messages in 2.450 ms 
2021-03-01T20:57:00.6273478Z 2021-0***-01T14:57:00.609-0600 INFO pool-28-thread-1 c.h.m.i.p.r.e.r.RedisEntityListener Finished notifying 1 messages in 104.7 μs 
2021-03-01T20:57:00.6275460Z 2021-0***-01T14:57:00.625-0600 INFO pool-28-thread-1 c.h.m.i.p.r.e.r.RedisEntityListener Finished notifying 1 messages in 1***7.9 μs 
2021-03-01T20:57:00.6380714Z [ERROR] Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.10*** s <<< FAILURE! - in com.hedera.mirror.importer.parser.record.entity.redis.RedisEntityListenerTest
2021-03-01T20:57:00.6396324Z [ERROR] com.hedera.mirror.importer.parser.record.entity.redis.RedisEntityListenerTest.onSlowPublish  Time elapsed: 0.061 s  <<< FAILURE!
2021-03-01T20:57:00.6402705Z org.opentest4j.AssertionFailedError: 
2021-03-01T20:57:00.6404171Z 
2021-03-01T20:57:00.6406740Z Expecting:
2021-03-01T20:57:00.6408375Z  <1>
2021-03-01T20:57:00.6410711Z to be equal to:
2021-03-01T20:57:00.6412149Z  <9>
2021-03-01T20:57:00.6414028Z but was not.
2021-03-01T20:57:00.6429405Z 	at com.hedera.mirror.importer.parser.record.entity.redis.RedisEntityListenerTest.onSlowPublish(RedisEntityListenerTest.java:105)
```

**Actual Behavior**

**Expected Behavior**


**Environment:**
- Java: [e.g. OpenJDK 11.0.4]
- Node: [e.g. v12.9.1-x86]
- OS: [e.g. Ubuntu 18.04]
- Version: [e.g. 1.0.1]

**Additional Context**

--- a/hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/entity/redis/RedisEntityListenerTest.java
+++ b/hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/entity/redis/RedisEntityListenerTest.java
@@ -51,6 +51,8 @@ import com.hedera.mirror.importer.parser.record.entity.EntityBatchSaveEvent;
 @ExtendWith(MockitoExtension.class)
 class RedisEntityListenerTest {
 
+    private static final long TIMEOUT_MILLIS = 2000L;
+
     @Mock
     private RedisOperations<String, StreamMessage> redisOperations;
 
@@ -100,13 +102,13 @@ class RedisEntityListenerTest {
 
         //then
         //Thread is blocked because queue is full, and publisher is blocked on first message.
-        verify(redisOperations, timeout(500).times(1))
+        verify(redisOperations, timeout(TIMEOUT_MILLIS).times(1))
                 .executePipelined(any(SessionCallback.class));
         assertThat(saveCount.get()).isEqualTo(redisProperties.getQueueCapacity() + 1);
 
         latch.countDown();
         //All messages should be queued and published
-        verify(redisOperations, timeout(500).times(redisProperties.getQueueCapacity() + 2))
+        verify(redisOperations, timeout(TIMEOUT_MILLIS).times(redisProperties.getQueueCapacity() + 2))
                 .executePipelined(any(SessionCallback.class));
         assertThat(saveCount.get()).isEqualTo(redisProperties.getQueueCapacity() + 2);
     }
@@ -120,21 +122,21 @@ class RedisEntityListenerTest {
         //submitAndSave two messages, verify publish logic called twice
         submitAndSave(topicMessage1);
         submitAndSave(topicMessage2);
-        verify(redisOperations, timeout(500).times(2))
+        verify(redisOperations, timeout(TIMEOUT_MILLIS).times(2))
                 .executePipelined(any(SessionCallback.class));
 
         //submitAndSave two duplicate messages, verify publish was not attempted
         Mockito.reset(redisOperations);
         submitAndSave(topicMessage1);
         submitAndSave(topicMessage2);
-        verify(redisOperations, timeout(500).times(0))
+        verify(redisOperations, timeout(TIMEOUT_MILLIS).times(0))
                 .executePipelined(any(SessionCallback.class));
 
         //submitAndSave third new unique message, verify publish called once.
         Mockito.reset(redisOperations);
         submitAndSave(topicMessage3);
         entityListener.onCleanup(new EntityBatchCleanupEvent(this));
-        verify(redisOperations, timeout(500).times(1))
+        verify(redisOperations, timeout(TIMEOUT_MILLIS).times(1))
                 .executePipelined(any(SessionCallback.class));
     }
 

